config/initializers/fast_gettext.rb

```diff
@@ -8,6 +8,15 @@
   gettext_options = %w(--sort-by-msgid --location --no-wrap)
   Rails.application.config.gettext_i18n_rails.msgmerge = gettext_options
   Rails.application.config.gettext_i18n_rails.xgettext = gettext_options + ["--add-comments=TRANSLATORS"]
+
+  unless Rails.env.test?
+    db_file  = Rails.root.join("config", "vmdb.yml.db")
+    template = Rails.root.join("config", "vmdb.tmpl.yml")
+    config_file = File.exist?(db_file) ? db_file : template
+    if YAML.load(File.read(config_file)).fetch_path(:ui, :mark_translated_strings)
```

Comment:
shouldn't this be just `VMDB::Config.new("vmdb").config.fetch_path(:ui, :mark_translated_strings)`?